src/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.item {
+  flex-grow: 1;
+  height: 300px;
+  width: 33%;
+  min-width: 300px;
+  margin: 0;
+  font-size: 100px;
+  line-height: 300px;
+  font-family: Arial;
+  text-align: center;
+  color: white;
+}
+
+.block_1 {
+  background-color: rgb(255, 0, 0);
+  order: 1;
+}
+
+.block_2 {
+  background-color: rgb(204, 0, 0);
+  order: 2;
+}
+
+.block_3 {
+  background-color: rgb(153, 0, 0);
+  order: 3;
```

Comment:
You don't need to provide order property to those elements, which order should be the same as in HTML.